src/main/java/io/jenkins/plugins/gitlabserverconfig/servers/GitLabServer.java

```diff
@@ -220,15 +220,20 @@ public static FormValidation doCheckServerUrl(@QueryParameter String value) {
             try {
                 new URL(value);
             } catch (MalformedURLException e) {
-                LOGGER.error("Incorrect server url - %s", value);
-                return FormValidation.error("Malformed GitLab url (%s)", e.getMessage());
+                LOGGER.error("Incorrect url - %s", value);
+                return FormValidation.error("Malformed url (%s)", e.getMessage());
             }
-            // TODO:[JENKINS-57747] Add support for GitLab Ultimate (self hosted) and Gold (saas)
             if (GITLAB_SERVER_URL.equals(value)) {
-                LOGGER.info("Server URL is fine - %s", value);
+                LOGGER.info("Community version of GitLab - %s", value);
+            }
+            GitLabApi gitLabApi = new GitLabApi(value, "");
+            try {
+                gitLabApi.getProjectApi().getProjects(1, 1);
+                return FormValidation.ok();
+            } catch (GitLabApiException e) {
+                LOGGER.info("Unable to validate serverUrl - %s", value);
```

Comment:
Logger levels are too high, info usually shows up in log files and console.
But I suggest you lower it to LogLevel Fine. So it is up to the Jenkins admin to add a logger https://wiki.jenkins.io/display/JENKINS/Logging

Reply:
Though this is on the form validation, so logging is redudanct.

Reply:
Okay will remove it.